tests/lean/Arrays.lean

```diff
@@ -349,7 +349,7 @@ def non_copyable_array : Result Unit :=
   take_array_t (Array.make AB 2#usize [ AB.A, AB.B ])
 
 /- [arrays::sum]: loop 0:
-   Source: 'tests/src/arrays.rs', lines 247:4-253:1 -/
+   Source: 'tests/src/arrays.rs', lines 245:0-245:28 -/
```

Comment:
That's surprising: looks like you're using the span of the function signature now instead of the span of the body?